nucleus/src/main/java/za/co/absa/subatomic/infrastructure/application/view/jpa/ApplicationHandler.java

```diff
@@ -46,7 +46,9 @@ void on(ApplicationCreated event) {
                 .createdBy(createdBy)
                 .build();
 
-        applicationRepository.save(applicationEntity);
+        applicationEntity = applicationRepository.save(applicationEntity);
+
+        projectEntity.getApplications().add(applicationEntity);
```

Comment:
Does this automatically get persisted to the view after the add?

Reply:
It does because it's in a `@Transactional` wrapped method. So anything you do to an entity will be committed once the method completes.

Reply:
Pretty handy.